src/store/state/state-model.ts

```diff
@@ -237,5 +237,6 @@ export const stateModel: StateModelIF = {
   },
   completingParty: null,
   parties: null,
+  resolutions: null,
```

Comment:
Or `[]`? Which is better?

Reply:
I think `[]` is better. what would you suggest?

Reply:
Agreed. See my response above. Resolving.